README.md

```diff
@@ -169,6 +173,35 @@ User.reindex
 Depending on how much data you have, reindexing may take a while but you only need to do it once.
 *ActiveStash will automatically index (and delete) data as it records are created, updated and deleted.*
 
+## Current limitations
+
+Presently, ActiveStash provides no means to update the schema of a CipherStash collection. Therefore if you need to make any changes to the Collection schema itself (by using the `stash_index` or `stash_match_all` helpers) you must drop your collection and recreate it.
+
+If your indexed model is called `User` for example, you should run the following commands:
+
+```sh
+$ rails active_stash:drop[User]
+$ rails active_stash:create
+$ rails active_stash:reindex[User]
+```
+
+Support for zero-downtime Collection schema changes and reindexing is being actively worked on and will be available soon.
+
+### When to Reindex Your Collection
+
+These are the rules for when you *must* re-index your collection:
+
+1. You have imported, deleted or updated data in the table that backs your ActiveStash model via some external mechanism, OR
+2. You have added or removed a string/text column from the table that backs your ActiveStash model *and* you are using a `dynamic_match` index in your model
+
+### When to Drop, Recreate and Reindex Your Collection
+
+This is the rule to determine when you *must* drop, recreate and reindex your collection:
+
+1. Whenever you modify one or more ActiveStash index definitions in your model
```

Comment:
So *technically* you only need to reindex if you modify the settings of an existing index, or add a new index.  If the only change that is the removal of an index from a model, a drop/create/index cycle isn't needed; we'll just keep updating an unused index (which isn't great, but doesn't actively *hurt* anything).  Dunno if dropping an index from a model is likely to happen often enough in the near future to make the caveat worthwhile, but I thought I'd mention it.

Reply:
I'll mention it in the README.